README.Rmd

```diff
@@ -1,7 +1,5 @@
 ---
-output:
-  github_document:
-    html_preview: false
+output: github_document
```

Comment:
If I remember correctly, won't this change create an extra HTML?

Reply:
Rendering it locally doesn't create an HTML file

Reply:
I render it and a html file was generated.

Reply:
My bad, I'll fix this